v20.2/performance-benchmarking-with-tpc-c-100k-warehouses.md

```diff
@@ -202,108 +209,24 @@ CockroachDB offers a pre-built `workload` binary for Linux that includes the TPC
-    ~~~ shell
-    $ ulimit -n 200500 && ./workload run tpcc \
-    --partitions 27 \
-    --warehouses 100000 \
-    --duration 1m \
-    --ramp 1ms \
-    "postgres://root@<address of any CockroachDB node>:26257?sslmode=disable"
-    ~~~
-
-3. Wait for up-replication and partitioning to finish.
-
-    This will likely take 10s of minutes. To watch the progress, go to the **Metrics > Queues > Replication Queue** graph in the Admin UI. Once the **Replication Queue** gets to `0` for all actions and stays there, you can move on to the next step.
-
-## Step 6. Re-start CockroachDB
-
-At the moment, running TPC-C against CockroachDB at this scale requires customizations that are not in the latest official release, so you'll need to stop the cluster, put a new binary with [these patches](https://github.com/cockroachdb/cockroach/commit/8af8c9e4a6399ee7672cac1e65b70a211cd63172) on the 81 VMs for CockroachDB, and then restart the cluster.
-
-1. SSH to the first VM where CockroachDB is running.
-
-2. Stop the `cockroach` process:
-
-    {% include copy-clipboard.html %}
-    ~~~ shell
-    $ cockroach quit --insecure
-    ~~~
-
-3. Rename the old binary:
-
-    {% include copy-clipboard.html %}
-    ~~~ shell
-    $ i="$(which cockroach)"; mv "$i" "$i"_old
-    ~~~
-
-4. Download the new binary, rename it, and copy it into the `PATH`:
-
-    {% include copy-clipboard.html %}
-    ~~~ shell
-    $ wget https://edge-binaries.cockroachdb.com/cockroach/cockroach.linux-gnu-amd64.9582884c143639d3acc85a7ba1a829fb5a5ae9dd ; chmod 755 cockroach.linux-gnu-amd64.9582884c143639d3acc85a7ba1a829fb5a5ae9dd
-    ~~~
-
-    {% include copy-clipboard.html %}
-    ~~~ shell
-    $ cp -i cockroach.linux-gnu-amd64.9582884c143639d3acc85a7ba1a829fb5a5ae9dd /usr/local/bin/cockroach
-    ~~~
-
-    If you get a permissions error, prefix the command with `sudo`.
+1. Wait for up-replication and partitioning to finish.  You will know when they have finished because both the number of *lease transfers* and *Raft snapshots* will go down to `0` and stay there.  Note that this will likely take 10s of minutes.
 
-5. Re-start the node, using the same [`cockroach start`](cockroach-start.html) command you used the first time you started the node:
-
-    {% include copy-clipboard.html %}
-    ~~~ shell
-    $ cockroach start \
-    --insecure \
-    --advertise-addr=<node1 internal address> \
-    --join=<node1 internal address>,<node2 internal address>,<node3 internal address> \
-    --cache=.25 \
-    --max-sql-memory=.25 \
-    --locality=rack=0 \
-    --background
-    ~~~
-
-6. Repeat steps 1 - 5 for the other 80 VMs for CockroachDB nodes. Each time, be sure to use the `cockroach start` command you used the first time you started the node.
+    To monitor the number of Lease transfers, open the [Admin UI](admin-ui-overview.html), select the **Queues** dashboard, hover over the **Replication Queue** graph, and check the **Leases Transferred / second** data point.
```

Comment:
This is my translation of the comment from the gdoc about where to look for lease transfers and snapshots in the Admin UI.   I'm pretty sure these are the right graphs to look at but please let me know if not, or if there is some other way y'all were checking this.